src/components/header-component/header-component.tsx

```diff
@@ -0,0 +1,15 @@
+import { HeaderTop } from "components/header-top/header-top";
```

Comment:
Where is `React` import? Does component renders correctly without it?

Reply:
> Where is `React` import? Does component renders correctly without it?
yep. Starting from React 17 it's not necessary to import React to use JSX.